submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+    border: 1px solid rgba(125, 125, 255, 0.5);
+    border-radius: 2px;
+}
+
+.item {
+    width: 58px;
+    height: 58px;
+    margin-bottom: 8px;
+    cursor: pointer;
+}
+
+.item-1 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1450px;
+}
+
+.item-2 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1314px;
+}
+
+.item-3 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -210px;
+}
+
+.item-4 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -554px;
+}
+
+.item-5 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -2347px;
+}
+
+.item-6 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -1036px;
+}
+
+.item-7 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -2000px;
+}
+
+.item-8 {
+    background-image: url(./img/allIcons.png);
+}
+
+.item-9 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -2070px;
+}
+
+.item-10 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -2210px;
+}
+
+.item-11 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -70px;
+}
+
+.item-12 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -140px;
+}
+
+.item-13 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -280px;
+}
+
+.item-14 {
+    background-image: url(./img/allIcons.png);
+    background-position: -3px -350px;
+}
+
+.item-15 {
+    background-image: url(./img/allIcons.png);
+    background-position: 0px -420px;
+}
+
+.item-16 {
+    background-image: url(./img/allIcons.png);
+    background-position: 0px -630px;
+}
+
+/* Draw hidden popup-ul-ul-ul */
+
+.pop-label::before {
+    display: none;
+    content: '';
+    position: absolute;
+    right: 56px;
+    top: 36px;
+    border: 10px solid transparent;
+    border-bottom: 10px solid #ccc;
+}
+
+.pop-label::after {
+    display: none;
+    content: '';
+    position: absolute;
+    right: 56px;
+    top: 38px;
+    z-index: 1;
+    border: 10px solid transparent;
+    border-bottom: 10px solid white;
+}
+
+.pop-check {
+    opacity: 0;
+    width: 24px;
+    height: 30px;
+}
+
+/* hide and show popup-ul */
+
+.pop-check:checked ~ .popup,
+.pop-check:checked ~ .pop-label::before,
+.pop-check:checked ~ .pop-label::after {
+    display: block;
+}
+
+/* more button */
+
+.more-pop {
+    height: 40px;
+}
+
+.more-button {
+    display: flex;
+    justify-content: center;
+    min-width: 320px;
+    width: 100%;
+    height: 40px;
+    text-align: center;
+    margin: 0;
+    align-content: center;
+    color: rgba(0, 0, 0, 0.87);
+    /* border: 1px solid red; */
+    cursor: pointer;
+    opacity: 0;
+    position: relative;
+    top: 20px;
+}
+
+.more-text {
+    font: small arial, sans-serif;
+    margin: auto 0;
+    top: -25px;
```

Comment:
Check duplicate properties in your css, you use top and left 2 times on this selector, need to fix.

Reply:
@CuteShaun done this.